itstool fails to build on amd64

The package itstool with version 2.0.7 and epoch 1 fails to build from source with the following error:

```
running step "fetch"
--2025-03-10 08:14:47--  https://github.com/itstool/itstool/archive/2.0.7/itstool-2.0.7.tar.gz
Resolving github.com... 140.82.114.3
Connecting to github.com|140.82.114.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/itstool/itstool/tar.gz/refs/tags/2.0.7 [following]
--2025-03-10 08:14:47--  https://codeload.github.com/itstool/itstool/tar.gz/refs/tags/2.0.7
Resolving codeload.github.com... 140.82.112.9
Connecting to codeload.github.com|140.82.112.9|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-gzip]
Saving to: 'itstool-2.0.7.tar.gz'

     0K .......... .......... .......... .......... .......... 1.08M
    50K .......... .......... ........                          825K=0.08s

2025-03-10 08:14:48 (984 KB/s) - 'itstool-2.0.7.tar.gz' saved [80082]

fetch: Expected sha256: fba78a37dc3535e4686c7f57407b97d03c676e3a57beac5fb2315162b0cc3176
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force 
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: /usr/bin/autoconf --force
autoreconf: configure.ac: not using Autoheader
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:2: installing './install-sh'
configure.ac:2: installing './missing'
autoreconf: Leaving directory '.'
checking for a BSD-compatible install... /usr/bin/install -c
checking whether sleep supports fractional seconds... yes
checking filesystem timestamp resolution... 2
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking xargs -n works... yes
checking whether /usr/bin/python3.12 version is >= 2.6... yes
checking for /usr/bin/python3.12 version... 3.12
checking for /usr/bin/python3.12 platform... linux
checking for GNU default /usr/bin/python3.12 prefix... ${prefix}
checking for GNU default /usr/bin/python3.12 exec_prefix... ${exec_prefix}
checking for /usr/bin/python3.12 script directory (pythondir)... ${PYTHON_PREFIX}/lib/python3.12/site-packages
checking for /usr/bin/python3.12 extension module directory (pyexecdir)... ${PYTHON_EXEC_PREFIX}/lib/python3.12/site-packages
checking for python module libxml2... not found
configure: error: Python module libxml2 is needed to run this package
failed to build package: unable to run package itstool pipeline: unable to run pipeline: exit status 1
```

--- a/itstool.yaml
+++ b/itstool.yaml
@@ -1,7 +1,7 @@
 package:
   name: itstool
   version: 2.0.7
-  epoch: 1
+  epoch: 2
   description: ITS-based XML translation tool
   copyright:
     - license: GPL-3.0-or-later
@@ -18,7 +18,7 @@ environment:
       - busybox
       - ca-certificates-bundle
       - libxml2-py3
-      - python-3.12
+      - python3
 
 pipeline:
   - uses: fetch
@@ -29,7 +29,7 @@ pipeline:
   - runs: autoreconf -vif
 
   - runs: |
-      PYTHON="/usr/bin/python3.12" ./configure \
+      PYTHON="/usr/bin/python3" ./configure \
         --host=${{host.triplet.gnu}} \
         --build=${{host.triplet.gnu}} \
         --prefix=/usr \
@@ -67,7 +67,7 @@ test:
         - diffutils
         - docbook-xml
         - gettext
-        - python3<3.13
+        - python3
   pipeline:
     - runs: |
         itstool --help

